source/localizable/analytics/ecommerce.html.md.erb

```diff
@@ -32,9 +32,11 @@ Name            | Type   | Required | Description
 `revenue`       | String | Yes      | The total revenue or grand total of the order. This value must include shipping and tax costs. Any additional payment cost should be excluded.
 `shipping`      | String | Yes      | The total shipping cost of the order. Any additional payment cost should be excluded.
 `tax`           | String | Yes      | The total tax of the order.
-`paid_by`       | String | No       | The payment method type of the order. The recommended payment method types are described in the table <%= link_to 'below', '', anchor: 'recommended-payment-method-types' %>.
+`paid_by`       | String | No<b><sup>1</sup></b> | The payment method type of the order. The recommended payment method types are described in the table <%= link_to 'below', '', anchor: 'recommended-payment-method-types' %>.
 `paid_by_descr` | String | No       | The description of payment method of the order that explains the payment method type (free text).
 
+<small><b><sup>1</sup></b> Required for all items that belong to Technology categories</small><br />
```

Comment:
Maybe we should update the message here. Someone may interpret it differently and send the paid_by/paid_by_descr fields only on orders that include technology products.

`Required for shops that have products in technology categories.`

Reply:
true, this needs to apply in all new comments we'll update (my2c, we can also make Technology start with a capital T since it's a title)